lib/engine.js

```diff
@@ -545,6 +574,13 @@ class Engine {
         }
         // }
 
+        // add final metrics (for activation metric)
+        this[INTERNAL].metrics.add({
+            // durations
+            processingTime: this[INTERNAL].timers.processingTime.currentDuration(),
+            downloadDuration: this[INTERNAL].timers.download.currentDuration(),
```

Comment:
for activation metrics, you should use `.totalDuration()` instead of `currentDuration()`: https://github.com/adobe/asset-compute-sdk/blob/master/lib/utils/timer.js#L61 which will sum up all the stop/starts throughout the pipeline
(see how we used it in the sdk: https://github.com/adobe/asset-compute-sdk/blob/master/lib/worker.js#L539-L548)